2018/CG-06-12.md

```diff
@@ -0,0 +1,45 @@
+![WebAssembly logo](/images/WebAssembly.png)
+
+## Agenda for the June 12 video call of WebAssembly's Community Group
+
+- **Where**: zoom.us
+- **When**: June 12, 4pm-5pm UTC (June 12, 9am-10am Pacific Time)
```

Comment:
The invite popped the 13th at 8am (UTC+2) in my calendar. Which one is correct?

Reply:
The meeting is on June 12th at 4pm UTC. I'd be fine including a time-zone converting link, though I'd prefer an ad-free one.